aptos-move/block-executor/src/code_cache_global.rs

```diff
@@ -64,88 +54,93 @@ where
         self.valid.load(Ordering::Acquire)
     }
 
-    /// Returns the module code stored is this [ImmutableModuleCode].
-    fn inner(&self) -> &Arc<ModuleCode<DC, VC, E, Option<TxnIndex>>> {
+    /// Returns the module code stored is this [Entry].
+    fn module_code(&self) -> &Arc<ModuleCode<DC, VC, E>> {
         self.module.deref()
     }
 }
 
-/// An immutable cache for verified code, that can be accessed concurrently thought the block, and
-/// only modified at block boundaries.
-pub struct ImmutableModuleCache<K, DC, VC, E> {
+/// A read-only module cache for verified code, that can be accessed concurrently within the block.
+/// Can only be modified safely at block boundaries.
+pub struct GlobalModuleCache<K, DC, VC, E> {
```

Comment:
I have trouble understanding what those generics stand for, maybe use a full word instead of abbr except for obvious thing like K?

Reply:
Updated generic names for `Entry` to full words